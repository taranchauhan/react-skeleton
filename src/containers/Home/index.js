import React from 'react';
import styled from 'styled-components';

const StyledDiv = styled.div`
  margin-top: 10px;
`;

const Home = () => (
  <StyledDiv>
    <h2>Home</h2>
  </StyledDiv>
);

export default Home;
