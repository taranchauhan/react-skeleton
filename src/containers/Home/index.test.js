import React from 'react';
import { shallow } from 'enzyme';
import Home from './index';

describe('<Home />', () => {
  it('should render the Home page', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
