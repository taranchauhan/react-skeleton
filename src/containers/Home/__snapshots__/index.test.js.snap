// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render the Home page 1`] = `
<styled.div>
  <h2>
    Home
  </h2>
</styled.div>
`;
